lib/rspecq/worker.rb

```diff
@@ -40,12 +40,19 @@ class Worker
     # Defaults to 3
     attr_accessor :max_requeues
 
+    # If set and not nil, it will force the run to stop after the designated
+    # number of failed specs.
+    #
+    # Defaults to nil
+    attr_accessor :fail_fast
```

Comment:
It'd be nice to have this as an `attr_accessor`, but this won't work right now, because `Queue` needs this passed in `Queue#initialize`. So unfortunately this needs to be moved back to an argument in `#initialize` (sorry).

But we can default it to `nil`, so that it's not required when initializing a worker.

Reply:
Is this still valid?

Reply:
No it's not.